src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -223,6 +224,15 @@ export class AeternityAdapter extends BaseProtocolAdapter {
     // TODO
   }
 
+  override async fetchTokenInfo(): Promise<IToken | undefined> {
+    // TODO if needed
+    return undefined;
+  }
+
+  override async transferToken() {
+    return undefined;
+  }
```

Comment:
I suggest to move the `transferToken` logic from `fungibleTokens` composable here as this logic strictly refers to AE.